pkgs/development/libraries/libunwind/default.nix

```diff
@@ -9,6 +9,12 @@ stdenv.mkDerivation rec {
     sha256 = "sha256-SmrsZmmR+0XQiJxErt6K1usQgHHDVU/N/2cfnJR5SXY=";
   };
 
+  # Fix for aarch64 and non-4K pages. Remove once upgraded past 1.6.2.
+  patches = lib.optional stdenv.isAarch64 (fetchpatch {
+    url = "https://github.com/libunwind/libunwind/pull/330.patch";
```

Comment:
this is not a reproducible url. someone could push to the pull request and change it

go to the latest commit in the pull and add `.patch` to the end

Reply:
Good point. thanks for noticing